lib/reference/atKeywords.mjs

```diff
@@ -7,6 +7,7 @@ export const nestingSupportedAtKeywords = new Set([
 	'media',
 	'scope',
 	'supports',
+	'starting-style',
```

Comment:
Id suggest to add `nestingSupportedAtKeywords` to l34 and remove the duplicates from it.
i.e. since `nestingSupportedAtKeywords` are always \@ keywords they are automatically part of `atKeywords`

Reply:
Make sense. Let's refactor `atKeywords.mjs` like this:

- Add `nestingSupportedAtKeywords` to `atKeywords`:
  ```diff
  -export const atKeywords = uniteSets(pageMarginAtKeywords, [
  +export const atKeywords = uniteSets(nestingSupportedAtKeywords, pageMarginAtKeywords, [
  ```
- Remove duplicate at-rules (e.g. `container`) from `atKeywords`:
  ```diff
  -	'container',
  ```

Ref: #7420 (original issue)